kernel/src/model/common.rs

```diff
@@ -2,7 +2,7 @@ use derive_builder::Builder;
 use serde::{Deserialize, Serialize};
 use std::fmt;
 
-#[derive(Deserialize, Debug, Serialize, PartialEq)]
+#[derive(Deserialize, Debug, Serialize, Eq, PartialEq)]
```

Comment:
Added `Eq` trait as per this [clippy lint](https://rust-lang.github.io/rust-clippy/master/index.html#/derive_partial_eq_without_eq).